CMakeLists.txt

```diff
@@ -153,8 +153,11 @@ ENDIF(APPLE)
 SET(H2_SYS_PATH "${CMAKE_INSTALL_PREFIX}/share/hydrogen")
 # TODO remove data from path, could be .config/hydrogen
 SET(H2_USR_PATH ".hydrogen")
-SET(H2_APP_ICON_PATH "/usr/share/pixmaps" CACHE STRING "Default system location of H2 application icon")
 
+#Changing this to user writable location will allow successful, non-root, deployment (e.g: make install).
+IF(NOT MINGW AND NOT APPLE)
```

Comment:
I don't think it's strictly necessary, since whole `INSTALL` block is enclosed in `IF(NOT MINGW AND NOT APPLE)` anyway.

Reply:
Well it is exactly the reason I put the definition of a variable under the same conditions it will be used. Also thought it derives directly from @mauser 's comment above regarding making it clear that is not used under mac and win.

Reply:
I'm not saying it's wrong. I think either way is OK.

Reply:
I'm sorry that i have caused some confusion here. What i meant that you can't tell from the name of that option that this option is only usable in a meaningful way on a linux system. But i guess most people which tinker with those option will have a look at the cmake script nontheless, so everything should be fine!